examples/example-next-static-export-sdk-2.x/src/lib/client.ts

```diff
@@ -30,6 +34,10 @@ export const getAuth = () => {
       clientId,
       url,
       redirectUrl,
+      true,
+      undefined,
+      window.location.toString(),
```

Comment:
We have to include these because of the order of parameters of the `createPublicOAuthClient` function

Reply:
I think this is exposing defaults that the user doesn't have to worry about. Should we look into perhaps introducing a new overload of createPublicOauthClient that takes an options dict with defaults? e.g:
function createPublicOauthClient({
  clientId,
  url,
  redirectUrl,
  x = true,
  y = 600,
  z = window.location.toString(),
  scopes = [],
}: OAuthClientOptions)

Reply:
This was what the older 1.x SDKs used. I would want to clarify why this was changed in 2.0

Reply:
Decided to keep it how it is given that the current SDK only support this method of client creation